CMakeLists.txt

```diff
@@ -3,6 +3,20 @@ cmake_minimum_required(VERSION 3.1)
 project(PodEmbedded VERSION 1.0
                     DESCRIPTION "Pod embedded")
 
+
+
+set(CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/CMakeFiles/modules)
+if(CMAKE_COMPILER_IS_GNUCXX)
```

Comment:
Couple implementation questions for you:

1) Do we always want to generate code coverage reports when we run UTs? Is it worth making a separate command?
2) If we do want to always run them together, consider combining the CMake conditionals for including GTest (at the bottom of this file) and codecov into one. Keeps complexity down.

Reply:
1. Yes we do - since we're testing the coverage of our UTs it should be tied together. I personally don't think it's worth making a separate command because it would always be tied to the UTs.
2. 👍